### ISSUE TITLE
MD087: Unused comments in backticks shouldn't be caught

### ISSUE THREAD

**Issue by eread** [rvben/rumdl#861]
### rumdl version

0.2.69

### Installation method

mise

### Bug description

Rule MD087 (https://github.com/rvben/rumdl/blob/993b5b10feca512e07d017ba4550fe690cd08c51/docs/md087.md) catches instances of comments that are used to disable rules that aren't acting on anything.

However, I'd expect such inline comments in backticks to be ignored, but they aren't.

### Steps to reproduce

1. Create a file called `test.md` with this content:
   ```markdown
   # Test Markdown
   
   These directives are in backticks, so shouldn't be caught by `rumdl`.
   
   `<!-- rumdl-disable MD001 -->`
   
   `<!-- markdownlint-disable MD001 -->`
   
   More test Markdown.
   ```
1. Run this command `rumdl check --enable MD001,MD087 test.md`.

### Expected behavior

I would expect no violations, because there are no "active" comments that disable rules.

### Actual behavior

Comments in backticks are treated as active. This is the result of running `rumdl check --enable MD001,MD087 test.md`:

```shell
test.md:5:2: [MD087] Unused disable comment: MD001
test.md:7:2: [MD087] Unused disable comment: MD001
```

### Configuration

None. Enable rules through the command line.

### Operating system

macOS

**Comment by rvben** [comment]
Thanks for the report, and for the copy-pasteable reproduction. That made this quick to confirm.

Fixed in **v0.2.70**. Your exact steps on the released build:

```
$ rumdl check --enable MD001,MD087 test.md
Success: No issues found in 1 file
```

And the control, the same two directives without backticks, so the rule is still doing its job:

```
ctl.md:3:1: [MD087] Unused disable comment: MD001
ctl.md:5:1: [MD087] Unused disable comment: MD001
```
